lib/improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -349,6 +350,11 @@ def __init__(self, distribution, desired_units,
         self.desired_units = desired_units
         self.predictor_of_mean_flag = predictor_of_mean_flag
         self.minimiser = ContinuousRankedProbabilityScoreMinimisers()
+        # Setting default values for coeff_names. Beta is the final
+        # coefficient name in the list, as there can potentially be
+        # multiple beta coefficients if the ensemble realizations, rather
+        # than the ensemble mean, are provided as the predictor.
+        self.coeff_names = ["gamma", "delta", "a", "beta"]
```

Comment:
Why is the third entry not "alpha"? The return statement of create_coefficients_cube suggests it should be.

Reply:
I've updated "a" to be "alpha" here and elsewhere in the unit tests, so as to avoid confusion.